include/deal.II/grid/grid_tools_cache.h

```diff
@@ -170,12 +182,15 @@ namespace GridTools
      * Therefore this function must be called by all processes
      * at the same time.
      *
-     * While each box may only cover part of a process's locally
-     * owned part of the triangulation, the boxes associated with
-     * each process jointly cover the entire local portion.
+     * The local bounding boxes are constructed by calling the function
+     * extract_rtree_level() applied to the object returned by
+     * get_locally_owned_cell_bounding_boxes_rtree(), with the specified level
+     * as input.
+     *
+     * @author Giovanni Alzetta, 2018, Luca Heltai, 2020
      */
     const RTree<std::pair<BoundingBox<spacedim>, unsigned int>> &
-    get_covering_rtree() const;
+    get_covering_rtree(const unsigned int level = 0) const;
```

Comment:
I don't think I understand what the level corresponds to. Maybe I need to look this up with `get_locally_owned_cell_bounding_boxes_rtree()` -- but actually, having done so, that function doesn't take a level.

Reply:
it is the `extract_rtree_level()` that takes a level as input...

Reply:
I see. Can you point that out in the documentation that that is what `level` refers to?

Reply:
I'm afraid that, after reading through the documentation of that function, I still don't know what the level of an rtree is :-(

Reply:
but that is well defined for an `rtree` object: see https://en.wikipedia.org/wiki/R-tree. And we do point to that page in the `RTree` documentation. :)

Reply:
I added a phrase to specify what we mean by level (and added a pointer to the wikipedia page).